### ISSUE TITLE
Ip anonymization 

### ISSUE THREAD

**Issue by GautierDig** [yireo/Yireo_GoogleTagManager2#255]
Hello, 

in order to be GDPR compliant, we would like to enable the IP anonymization. I couldn't find if that was implemented in your module. Does the module Yireo_GoogleTagManager2 can anonymize IPs ?

**Comment by jissereitsma** [comment]
No, this module does not anonymize IP and it actually does not handle IPs whatsoever. Are you sure this should be part of an extension? Because all the tutorials I can find are explaining how to do this in the GTM console, not in an extension?

**Comment by GautierDig** [comment]
Hi @jissereitsma,

Thanks for your quick feedbacks. Yes indeed that also what I think. I will check on my side. Thanks !
